docs/source/interfaces/Indirect Settings.rst

```diff
@@ -0,0 +1,326 @@
+
+OK
+  This will apply the changes made to the interface and then exits the settings window.
+
+Cancel
+  This disregards any changes which have not been applyed and exits the window.
+
+
+General
+-------
+
+This section contains options which will be applyed across all of the Indirect interfaces.
+
+Options
+~~~~~~~
+
+Facility
+  Allows you to choose the selected facility.
+
+
+Indirect Bayes
+--------------
+
+This section is displayed only when the settings window is opened from the Bayes
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Analysis
+-------------
+
+This section is displayed only when the settings window is opened from the Data Analysis
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Corrections
+----------------
+
+This section is displayed only when the settings window is opened from the Data Corrections
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Reduction
+--------------
+
+This section is displayed only when the settings window is opened from the Data Reduction
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Other Interfaces
+----------------
+
+There are currently no interface-specific settings for the Diffraction, Simulation and Tools
+interfaces.
+
+
+Glossary of Allowed Suffixes
+----------------------------
+
+_red
```

Comment:
That's a good piece of documentation. But what I meant is to also include a description about the axes and units for each tag. It can be that someone has some processed workspace that loads into mantid, and then needs to figure out what the suffix should be to obey the semantics. Especially for the analysis part, we can imagine that some users might want to fit data that are not even reduced with mantid.